deps/build.jl

```diff
@@ -138,7 +138,9 @@ function main()
     # discover other CUDA toolkit artifacts
     ## required
     config[:libdevice] = find_libdevice(config[:target_support], toolkit_dirs)
-    config[:libdevice] == nothing && build_error("Available CUDA toolchain does not provide libdevice")
+    config[:libdevice] === nothing && build_error("Available CUDA toolchain does not provide libdevice")
+    config[:libcudadevrt] = find_libcudadevrt(unique(toolkit_dirs))
```

Comment:
@maleadt how should `find_libcudadevrt` handle multiple dirs?

Reply:
CI fails because of:
```
┌ Error: Error building `CUDAnative`: 
│ Available CUDA toolchain does not provide libcudadevrt.
```

probably because the logic I added to find libcudadevrt is to fragile.

Reply:
look in each one of them. Multiple dirs indicate the CUDA installation is spread across the system (as it happens with many Linux distros who feel like repackaging stuff), so we need to deal with that.

Reply:
```
$ docker run -it --runtime=nvidia --rm juliagpu/julia:dev-cuda /bin/bash                                                                                                                                                     
root@2eaa02354be3:/# find / | grep libcudadevrt
/usr/local/cuda-10.0/targets/x86_64-linux/lib/libcudadevrt.a
```